flexbox/src/main/java/com/google/android/flexbox/FlexboxHelper.java

```diff
@@ -815,6 +769,7 @@ private void checkSizeConstraints(View view, int index) {
             int widthSpec = View.MeasureSpec.makeMeasureSpec(childWidth, View.MeasureSpec.EXACTLY);
             int heightSpec = View.MeasureSpec
                     .makeMeasureSpec(childHeight, View.MeasureSpec.EXACTLY);
+//            mFlexContainer.measureChildFlexItem(view, widthSpec, heightSpec);
```

Comment:
Oops, sorry the method was deleted from the interface but forgot to delete these lines.
Done.